packages/manager/modules/pci/src/projects/project/storages/databases/components/plans-list/plans-list.html

```diff
@@ -0,0 +1,83 @@
+<div class="row">
+    <oui-select-picker
+        class="d-inline-block col-md-6 col-xl-4 my-3"
+        data-ng-repeat="plan in $ctrl.plans track by plan.name"
+        data-disabled="$ctrl.currentPlan && $ctrl.plans.indexOf($ctrl.currentPlan) >= $index"
+        data-name="{{:: plan.name }}"
+        data-model="$ctrl.selectedPlan"
+        data-label="{{:: $ctrl.capitalize(plan.name) }}"
+        data-values="[plan]"
+        data-variant="light"
+        data-on-change="$ctrl.onChange({ selectedPlan: modelValue })"
+        data-required
+    >
+        <oui-select-picker-section>
+            <div class="d-flex">
+                <span
+                    data-translate="pci_database_plans_list_spec_ram"
+                    data-translate-values="{ 
+                        min: (plan.minMemory | cucBytes:0:false:'GB'),
+                        max: (plan.maxMemory | cucBytes:0:false:'GB')
+                    }"
+                ></span>
+            </div>
```

Comment:
Not sure why the div.d-flex is needed here, maybe we should just have `block` elements

Reply:
removed###